src/libprojectM/CMakeLists.txt

```diff
@@ -12,11 +11,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
     add_compile_definitions(
             NOMINMAX
             WIN32_LEAN_AND_MEAN
-            USE_FTGL
```

Comment:
This define is used in emscripten builds to use ftgl.h in the OpenGL iincludes. Please keep it, otherwise it'll break stuff.

Reply:
Removed as per comments below.